### ISSUE TITLE
Lemonbar doesn't always show the input we feed it

### ISSUE THREAD

**Issue by domsson** [domsson/succade#29]
Because of [a bug in lemonbar](https://github.com/LemonBoy/bar/issues/107), feeding new input to lemonbar in quick succession doesn't work, which can lead to lemonbar showing a previous state rather than the newest.

Example: Assume succade is configured with two blocks, both static, one that produces output ever so slightly faster than the other. This will cause succade to feed lemonbar two times in quick succession, the first time only containing the faster block, the second time containing both blocks. However, lemonbar will show only the first block, as it completely ignored the second input, as it came in too fast.

We can't really fix this in succade, so there are two options here:
- fork lemonbar, fix it, send a PR (however, there are many unanswered issued and unmerged PRs already, not sure if the maintainer is still active on the project)
- find some kind of workaround in succade that can optionally be activated via command line switch (for example, a minimum time between feeds)

**Comment by shdown** [comment]
See https://github.com/shdown/bar/commit/eb0e9e79b9a35c3093f6e2e90d9fbd175a2305ef

**Comment by domsson** [comment]
@shdown thanks for making me aware. Did you turn that into a PR for lemonbar?

**Comment by shdown** [comment]
Yes, but it went nowhere: https://github.com/LemonBoy/bar/pull/198

**Comment by domsson** [comment]
Looks like it was merged now!
